src/behavior.ts

```diff
@@ -24,66 +24,186 @@
  *  THE SOFTWARE.
  */
 
-import * as d3 from "d3";
-
 import { getOpacity } from "./utils";
 import { DotPlotDataGroup } from "./dataInterfaces";
 
 // d3
 import { Selection } from "d3-selection";
```

Comment:
lets keep d3 beginning naming for d3 methods.
import `{ Selection as d3Selection }` from "d3-selection";

Please refactor in the same way other d3 imports.

Reply:
Done.